avatars/api.py

```diff
@@ -1037,4 +1194,119 @@ def avatarization_pipeline_with_processors(
         )
 
 
+def upload_batch_and_get_order(
+    client: "ApiClient",
+    training: pd.DataFrame,
+    splits: List[pd.DataFrame],
+    timeout: int = 10,
```

Comment:
```suggestion
    timeout: int = DEFAULT_TIMEOUT,
```
I think there is a variable called like this, either in `api.py` or `client.py`